recipes/default.rb

```diff
@@ -1,22 +1,8 @@
-include_recipe 'chef_handler'
-
-# install inspec
-inspec 'inspec' do
+chef_gem 'inspec' do
   version node['audit']['inspec_version']
```

Comment:
Any reason why we are not using `version new_resource.version if new_resource.version != 'latest'`?